src/js/views/patients/patient/dashboard/flow-item.hbs

```diff
@@ -3,7 +3,7 @@
   <span class="patient__action-name">{{ name }}</span>
 </td>
 <td class="table-list__cell patient__list-meta">
-  <span class="table-list__meta">{{~ remove_whitespace ~}}
+  <span class="table-list__meta u-margin--l-32">{{~ remove_whitespace ~}}
```

Comment:
There can be both actions and flows in the dashboard/arcive lists.

So I added this to keep the flows aligned with the actions. Which will now have the details icon that has 32px width.